### DISCUSSION TITLE
Version 1.10 Windows won't install

### DISCUSSION THREAD

**Discussion by ak1dd** [exult/exult#554] [General]
After selecting game folders, ready to install, install:  "Download Failed: 404 Not Found"

I used all default options.  I did nothing other than change my game directories to the GOG Galaxy directories.  Otherwise just pushed next, next, next...

One time it said something about Exult_Audio_Installer.exe not found but every time afterward it just fails with a 404 error.

**Comment by DominusExult** [comment]
in the options of the installer deselect "Download and install" - these are all optional downloads. If they don't work it's likely a problem on your end or rather a problem of our installer with your internet connection.
But I'll investigate

**Comment by DominusExult** [comment]
> One time it said something about Exult_Audio_Installer.exe not found but every time afterward it just fails with a 404 error.

^^^of that I would like a screenshot as the Exult installer does not touch the Exult Audi Installer. It will download the zip of our audio files and install these without downloading or running that installer.

**Comment by wench** [comment]
There is a problem here. The 1.10 release here on GitHub is missing the assets for  sifixes, keyring and SFisland that the installer wants to download and will 404 trying to download those. The installer tries to download the assets from the latest release, so this will correct itself when the next snapshot is generated in about 12 hours time

**Comment by DominusExult** [comment]
ouch, I didn't think of this. I'm running the snapshot workflow now so it generates a snapshot release

**Comment by DominusExult** [comment]
This works again now. I've made a note in the release check list.

**Comment by wench** [comment]
I made a new issue related to this #556

**Comment by ak1dd** [comment]
Glad it was resolved.  Was certain it wasn't on my end or a clean install with a working net connection didn't work.  It was a default selection even if it is optional.

**Comment by DominusExult** [comment]
Yes, I'm sorry that you received the developer knee-jerk reaction from me ("Must be a user issue"). I try to be better but in this case I was so sure it works because it worked on all my test runs. Of course this fell apart when I tagged a new release :)
So thanks a lot for reporting this right away and again sorry!

**Comment by ak1dd** [comment]
No worries.  I *greatly* appreciate the work you do.  U7 is one of the most important/nostalgic titles in my life.  You've given me a lot more than a tiny amount of grief...  Thanks again.
